dataframe/arrow.go

```diff
@@ -179,3 +181,75 @@ func fieldToArrow(f FieldType) (arrow.DataType, error) {
 		return nil, fmt.Errorf("unsupported type: %s", f)
 	}
 }
+
+// UnMarshalArrow converts a byte representation of an arrow table to a Frame
+func UnMarshalArrow(b []byte) (*Frame, error) {
+	fB := filebuffer.New(b)
+	fR, err := ipc.NewFileReader(fB)
+	defer fR.Close()
+	if err != nil {
+		return nil, err
+	}
+	schema := fR.Schema()
+	metaData := schema.Metadata()
+	frame := &Frame{}
+	if nameIdx := metaData.FindKey("name"); nameIdx > 0 {
+		frame.Name = metaData.Values()[nameIdx]
+	}
+	if refIdx := metaData.FindKey("refId"); refIdx > 0 {
+		frame.RefID = metaData.Values()[refIdx]
+	}
+	// TODO Labels
```

Comment:
TODO: Labels are ignored, need Labels to have a fromstring or some such method, fetch them from the metadata, and put them in `frame` object.